build.cake

```diff
@@ -138,133 +128,45 @@ Task("Clean")
+            msbuildSettings.ToolVersion = MSBuildToolVersion.VS2017;
+        else
+            msbuildSettings.ToolPath = Context.Tools.Resolve("msbuild");
 
-Task("BuildNetStandard13")
-    .Description("Builds the .NET Standard 1.3 version of the framework")
-    .WithCriteria(IsRunningOnWindows())
-    .Does(() =>
-    {
-        if(!isDotNetCoreInstalled)
-        {
-            Warning(".NET Standard 1.3 was not built because .NET Core SDK is not installed");
-            return;
-        }
-        BuildProject("src/NUnitFramework/framework/nunit.framework-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/nunitlite/nunitlite-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/mock-assembly/mock-assembly-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/testdata/nunit.testdata-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/slow-tests/slow-nunit-tests-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/tests/nunit.framework.tests-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/nunitlite.tests/nunitlite.tests-netstandard13.csproj", configuration);
-        BuildProject("src/NUnitFramework/nunitlite-runner/nunitlite-runner-netstandard13.csproj", configuration);
-    });
+        MSBuild(SOLUTION_FILE, msbuildSettings);
 
-Task("BuildNetStandard16")
-    .Description("Builds the .NET Standard 1.6 version of the framework")
-    .WithCriteria(IsRunningOnWindows())
-    .Does(() =>
-    {
-        if(!isDotNetCoreInstalled)
+        DotNetCorePublish("src/NUnitFramework/tests/nunit.framework.tests.csproj", new DotNetCorePublishSettings
         {
-            Warning(".NET Standard 1.6 was not built because .NET Core SDK is not installed");
-            return;
-        }
-        BuildProject("src/NUnitFramework/framework/nunit.framework-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/nunitlite/nunitlite-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/mock-assembly/mock-assembly-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/testdata/nunit.testdata-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/slow-tests/slow-nunit-tests-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/tests/nunit.framework.tests-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/nunitlite.tests/nunitlite.tests-netstandard16.csproj", configuration);
-        BuildProject("src/NUnitFramework/nunitlite-runner/nunitlite-runner-netstandard16.csproj", configuration);
+            Framework = "netcoreapp1.1",
+            Configuration = configuration,
+            OutputDirectory = BIN_DIR + "netcoreapp1.1/"
+        });
     });
 
 //////////////////////////////////////////////////////////////////////
```

Comment:
Unrelated, but the two `Test` and `Test Framework` headers here seem out of place, and the CheckForError task - mind tidying up while we're here? :-)

Reply:
Ah, nice. Rob, do you want to handle fixing the headers or should I?
Btw Chris your comments are all on the line above rather than the line below so I have to follow the link to the full changes to see what you're commenting on. 😆

Reply:
Sorry - because the lines I’m commenting on are unchanged this seemed to be as close as I could get them! 😞

Reply:
Ah, good point!